vmm/src/memory_manager.rs

```diff
@@ -1857,6 +1901,11 @@ impl MemoryManager {
         self.guest_ram_mappings.len() as u32
     }
 
+    #[cfg(target_arch = "aarch64")]
+    pub fn get_uefi_flash(&self) -> GuestMemoryAtomic<GuestMemoryMmap> {
```

Comment:
Rust coding style doesn't use `get_`

Reply:
Also, can it be `pub(crate)` instead?

Reply:
This method is only used in "load_firmware" for now. We can change it to crate if it is referenced in more places.